dist/dicom/src/structuredReportToHtml.ts

```diff
@@ -101,10 +105,14 @@ async function structuredReportToHtml(
     args.push('--add-document-type')
   }
   if (options.cssReference) {
-    args.push('--css-reference', '2')
+    const inputCountString = ('TextStream').includes('File') ? 'file' + inputs.length.toString() : inputs.length.toString()
+    args.push('--css-reference', inputCountString)
+    inputs.push({ type: InterfaceTypes.TextStream, data: { data: options.cssReference, path: ('TextStream').includes('File') ? inputCountString : '' } })
   }
   if (options.cssFile) {
-    args.push('--css-file', '3')
+    const inputCountString = ('TextFile').includes('File') ? 'file' + inputs.length.toString() : inputs.length.toString()
```

Comment:
` ('TextStream').includes('File')` and `('TextFile').includes('File')` are "static" -- could they be evaluated in `bindgen` execution so they do not need to be output and evaluated here?